roles/ceph-mon/tasks/ceph_keys.yml

```diff
@@ -0,0 +1,41 @@
+---
+# NOTE (leseb): wait for mon discovery and quorum resolution
+# the admin key is not instantanely created so we have to wait a bit
+- name: wait for client.admin key exists
+  wait_for: path=/etc/ceph/ceph.client.admin.keyring
+
+- include: openstack_config.yml
+  when:
+    openstack_config and
+    cephx
+
+- name: find ceph keys
+  shell: ls -1 /etc/ceph/*.keyring
+  changed_when: false
+  register: ceph_keys
+  when: cephx
+
+- name: set keys permissions
+  file: path={{ item }}
+        mode=0600
+        owner=root
+        group=root
+  with_items:
+    - "{{ ceph_keys.stdout_lines }}"
+
+- name: copy keys to the ansible server
+  fetch: src={{ item }}
+         dest=fetch/{{ fsid }}/{{ item }}
+         flat=yes
+  with_items:
+    - "{{ ceph_keys.stdout_lines }}"
+    - /var/lib/ceph/bootstrap-osd/ceph.keyring
+  when: cephx
```

Comment:
Will remove this task, since it's a backup feature for the community's ceph ansible.